scanner/src/routes/+page.svelte

```diff
@@ -181,16 +186,35 @@
 
     return true
   }
+
+  
+  onMount(() => {
+    startScan();
+  })
```

Comment:
There is supposed to be a screen that asks for you to tap on a "Turn on" button, because on the web you cannot start the QR code scanner without user interaction.

Reply:
It works on web by this without user interaction

Reply:
How incredibly strange, I tested this the very same way earlier and I got some security warnings from my browser. I think I'll have to test again tomorrow, maybe there's some specific browsers or circumstances where it doesn't.

Reply:
Added the first screen to Turn On camera.

Reply:
So I tested restoring the `onMount()`, and it is indeed blocked:

![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/834897b3-f2e9-4158-b856-15a745f9d7d1)

![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/75a3030c-6c7c-4f3f-bd4d-c7215369fed7)

Putting the `Turn on` button back makes it work fine.